### ISSUE TITLE
Install blocked by administrator

### ISSUE THREAD

**Issue by PeteLawler** [nE0sIghT/chrome-gnome-shell-mirror#11]
I can't seem to install this from the Google Store. Error message is:
GNOME Shell integration (extension ID "gphhapmejobijbbhgpjhcjognlahblep") is blocked by the administrator.

I am running Chromium as my user, but I do have sudo/root access to change any file permissions required for debugging purposes.

Running GNOME  3.21.90 (Debian Stretch), Chromium 52.0.2743.116 (also from Debian Stretch)

**Comment by nE0sIghT** [comment]
Hi,

1. Try to remove 2 files:
/etc/opt/chrome/policies/managed/chrome-gnome-shell.json
/etc/chromium/policies/managed/chrome-gnome-shell.json

2. Uninstall extension from "Extensions page"
3. Reinstall ppa package

**Comment by nE0sIghT** [comment]
I added this information to [Troubleshooting section of Installation Guide](https://wiki.gnome.org/Projects/GnomeShellIntegrationForChrome/Installation).
Reopen if you still experience problems.

**Comment by PeteLawler** [comment]
> 3. Reinstall ppa package

Ah, I think I see where the issue is coming from. As I'm on Debian Stretch, I'm not using a PPA but using Git. The instructions on https://wiki.gnome.org/Projects/GnomeShellIntegrationForChrome/Installation include step 3:
3. sudo make install

This creates the two files you mention, but with mode 644. I'm not eager to world writable that file. Anyway, it'd seem that if I rm /etc/chromium/policies/managed/chrome-gnome-shell.json (no idea about the /etc/opt/chrome...) I can install the extension just fine. I don't know Chrom(e|ium) at all, but seems perhaps that file is either not needed ... or, something else is a little wonky.

tl;dr - On Debian, not Ubuntu. No PPA install. After following manual install of connector from git, removing one of the files you mention gets me able to install the extension.

(Also, seems GIthub won't let me re-open... moving on)

**Comment by nE0sIghT** [comment]
It's some rare conflict between Chrome poilcy and already installed extension.
You just need remove policies, uninstall extension manually from browser.
And then return policies back - extension will be autoinstalled normally

**Comment by PeteLawler** [comment]
If I remove /etc/chromium/policies/managed/chrome-gnome-shell.json and uninstall the extension manually, then return the policy file the extension does not auto install.
Only by not having the /etc/chromium/policies/managed/chrome-gnome-shell.json file does it install. If I put this file back, the extension is auto-uninstalled.
So instead of trying to install the extension via the Chrome Store, I tried to load via unpacked/developer method via the git tree that I had installed the native connector from, but this also did not work.

"Failed to load extension from: ~/Downloads/chrome-gnome-shell/extension
Value 'key' is missing or invalid."

starting to suspect I may be better off blowing away my entire chromium settings, however this won't necessarily help the next person facing these issues...

**Comment by nE0sIghT** [comment]
> If I remove /etc/chromium/policies/managed/chrome-gnome-shell.json and uninstall the extension manually, then return the policy file the extension does not auto install.

And you receiving same "blocked by administrator" error?

> Value 'key' is missing or invalid

You can not load unpacked extension sources that way. You should build extension with `-DBUILD_EXTENSION=ON` option and then load extension from `build/extension/chrome` folder.

**Comment by rickysarraf** [comment]
I have the same problem that has been mentioned in this report. The moment you add back the policies/managed/chrome-gnome-shell.json file , chromium uninstalls the extension.

As for auto installation of the extension per policy file's presence, no, that did not happen either.

**Comment by rickysarraf** [comment]
![image](https://cloud.githubusercontent.com/assets/65480/18910817/4ecc3514-8597-11e6-8611-e9c1371df968.png)

**Comment by nE0sIghT** [comment]
> I have the same problem that has been mentioned in this report. 

Something is broken in your Chromium profile. You can confirm this by creating new clean Chromium profile.
Did you installed unpacked extension by hand some time ago?

Removing policies and uninstalling extension manually before returning policies did not fix this issue for you?

**Comment by rickysarraf** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On Wed, 2016-09-28 at 08:09 -0700, Yuri Konotopov wrote:
> Something is broken in your Chromium profile. You can confirm this by creating
> new clean Chromium profile.

I'll try and report that soon.

> Did you installed unpacked extension by hand some time ago?

Nope. Prior to today, I've used your .deb package from the PPA archive.

> Removing policies and uninstalling extension manually before returning
> policies did not fix this issue for you?

Just the presence of the policy file triggers chromium to autoremove the
extension (it pops-up but doesn't wait for user confirmation, more like just a
momentary notification).

If the policy file is present and you start chromium, it does not auto-install
the extension.

- -- 
Ritesh Raj Sarraf
RESEARCHUT - http://www.researchut.com
"Necessity is the mother of invention."
-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJX69+oAAoJEKY6WKPy4XVpq/EP/2vKJrQeOn4WgNG0FCrjOhRF
LsTw+lAusENQBIRof4jdKbadNpVg4OpwK/irWKFJ6VUpXjkeqdwNwIzWBf0DKdz6
9dLXaIE7eoo8PBwu0sxsmQYE/6vhUHmoQzj3a/ICd6jPzNuibxXyWbwPXaQ/qltB
qaeHnNG4pa34ecvr3Ev05ML7o/vwPptIhumQXffBc3pLhf5jNf9WVCjcRXDRAqHd
cJkVkc8AhbZ+q09jp3E5bP9dhL+7dlamZrgS2zjNHVxzSxz7QKykqlI7eQs8dhBP
PblkaQaRHGYLXCOLn+cmi1YwsyWxKyKFqDdQ3k2Fb2R7xv8/Cli9slsTpS+DpuMu
tk1nrBf2rmHK9u/nbZz/Ump+Mf8HLy8MX40Zj05CEcBWCBDnfA/gyHfaVO+r5obJ
p5zXIbFrCD1EX848QDfKak5Hn1shTB/qgma4puPmQ9g08wURBxvxm4AobsgDFutF
HOXPjw1g4nMgyLp2SI+DNq1H+l5YXkVkTaonsJ2Tjy2iXVvAfKeg9AnMyhxWpvs7
mAp2SY0KLqzb/P/PnOQy6aylIuZCmaLbKHtpJ9Eg89+zlIXpt2TQT/j4eqOIEs1e
4+B70H8LxIdEP2MRNzd5GVxAIB9mxGHsShv4FwbCZTcur8ieHuGWZKzI/NGcDJ8u
uzDILMl17z4m+S+B3rEz
=U/sj
-----END PGP SIGNATURE-----

**Comment by nE0sIghT** [comment]
> Just the presence of the policy file triggers chromium to autoremove the
extension

But if you remove policies extension comes back?

If so, try to remove it by hand and return policies.

**Comment by rickysarraf** [comment]
I can now confirm that if may not have anything to do with the chromium profile. I tried on a different a/c on the same machine.


![screenshot from 2016-09-28 20-54-02](https://cloud.githubusercontent.com/assets/65480/18920151/29925d42-85be-11e6-9f9b-b6b39951aa49.png)

**Comment by rickysarraf** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On Wed, 2016-09-28 at 08:27 -0700, Yuri Konotopov wrote:
> But if you remove policies extension comes back?

No. Chromium only auto-uninstalls the extension. I'm not seeing any auto-
install.

> If so, try to remove it by hand and return policies.
> 

I'm not sure what you mean by "return policies".

Removing the policy file by hand, allows, to manually install the extension from
the Chrome web store.

- -- 
Ritesh Raj Sarraf
RESEARCHUT - http://www.researchut.com
"Necessity is the mother of invention."
-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJX6+JLAAoJEKY6WKPy4XVpkmMP/2F8fzKqz7lXKXunSOt7dowW
oJ9xXlxfGBQuyqoBZHFwC2PjbPM2HU20s38y3TBMe/O4EyPSShMPJs+6M5e8Fu6I
sWzg3YhTXiMscFmeMY1w+DvKWUfFUZZVUxd/BZUEI51DaY/6h2D96rJl/aW24dzs
mQJwIkPBxgjHdhE2sHWIfYYPF20c0fylOD9iYMn9vhy5H3/PvOmZ99q0AhQ24XMZ
IJlEazlayMzFMg53q4Ke/THe5lY4RgjxPR97wBQzAbmjjX9DNGhHamv+CM0+fbRS
22cqs2k+4LLRRuH2ID4nNYwJJXBnAIJLqph4JdEkZSiyj2CgQUy2bc3LvRbsIoDu
Ey7vU+STz5JNTIQDeo/6RTBk5TdlGrpnlIPg5SbTIgNT0fM4BtBy2+AYWzWslZLE
KsxGfVf5laeH5h8+EW16tS9hq9EZarRKxcsm3l3mU3yOjAkVCxqgdgBeBuwoP53a
h+Z/jJJU5p3Du4OYywJ1/9E6CTgSNs3pxSnkYmYOhqVICoHlpsMf6O1fBrrW/RRg
yHtg+Udbm41UWcjPhEnUzzPWXyVaG2WNyeAGPrdLHfzEM8kgB5qmnEumKhnxXtW8
mczHNvjGbUbAO5pElQpDqOq8VO3ekYbxKzFnRyPE793nF3gEB2eUbJUU6ozWhwrq
oS/N8NzAENYpMk243Lgm
=MjMl
-----END PGP SIGNATURE-----

**Comment by nE0sIghT** [comment]
> I'm not sure what you mean by "return policies".

I faced this issue several times - one time with chrome-gnome-shell and multiple times in my organization while testing managed policies for other extensions. 

Here is what I experienced (using chrome-gnome-shell as example):

1. chrome-gnome-shell was installed in my Chromium profile from Chrome store manually.
2. I deployed policy file (/etc/chromium/policies/managed/chrome-gnome-shell.json) - chrome-gnome-shell was "visually" uninstalled from my Chromium profile and I get "Blocked by administrator" message.
3. Then I removed policy file - previously manually installed extension returned to my profile.
4. I removed installed extension from Chromium and deployed policy file again - extension auto installed from Chrome store as it should.

**Comment by rickysarraf** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On Wed, 2016-09-28 at 08:50 -0700, Yuri Konotopov wrote:
> Here is what I experienced (using chrome-gnome-shell as example):
> 

> chrome-gnome-shell was installed in my Chromium profile from Chrome store
> manually.

Yes. This was the case so far for me too. I installed the .deb (0.61) from your
PPA and the extenstion manually from Web Store.

> I deployed policy file (/etc/chromium/policies/managed/chrome-gnome-
> shell.json) - chrome-gnome-shell was "visually" uninstalled from my Chromium
> profile and I get "Blocked by administrator" message.

That's what was my initial suspicion. To verify that, I looked into the
extension folder, but couldn't find any sign of your package. Just that we are
on the same page, I've included the path here.

/home/rrs/.config/chromium/Default/Extensions/

So here's something new I noticed here. With 7.1, yes, the auto-uninstallation
was just actually disabling/hiding the extension from the browser.

Extensions/gphhapmejobijbbhgpjhcjognlahblep
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/extension.js
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/options.js
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/content-script-start.js
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/icons
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/icons/GnomeLogo-16.png
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/icons/GnomeLogo-48.png
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/icons/GnomeLogo-128.png
Extensions/gphhapmejobijbbhgpjhcjognlahblep/7.1_0/extension.html


> Then I removed policy file - previously manually installed extension returned
> to my profile.

Yes. Now this too happened.

> I removed installed extension from Chromium and deployed policy file again -
> extension auto installed from Chrome store as it should.

This is where things are failing. After removing the extension and deploying the
policy file back, the extension has not auto-installed so far.

Manually checking the folder also did not yield any hidden extension artifact.

rrs@learner:~/.config/chromium/Default/Extensions$ find . | grep -i
gphhapmejobijbbhgpjhcjognlahblep
2016-09-28 / 21:49:29 ♒♒♒  ☹  => 1  

rrs@learner:~/.config/chromium/Default/Extensions$ ls
/etc/chromium/policies/managed/
chrome-gnome-shell.json
2016-09-28 / 21:49:54 ♒♒♒  ☺  



Is there any action/flag required in the chromium browser to get it to honor alternative extensions ?
I have been reading up here: https://developer.chrome.com/extensions/external_extensions#preferences
but I don't see any special settings in the browser settings to be required.


- -- 
Ritesh Raj Sarraf
RESEARCHUT - http://www.researchut.com
"Necessity is the mother of invention."
-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJX6+5eAAoJEKY6WKPy4XVpEKcQAITKfcPzyjjeMZNK6KY+is6R
cZwJ4IYe9bd1OXQWoiPLG3tTVNkQGq0Wpjesxu2jO4zheKAdIutJMnyW40cLV9KY
WSTpIBWJMtWRXntmN9NsjuZMndn+YJ/67HTcIzAEQTONF5tfplBSonel5JlArX+V
yL418axRpFuldidpVFb9KUaIRvt3w6yKSSImItBHMA+JKCEDGlnlX1pFRKKB6Mnm
648BtvphPQP92bIvGG1ZR60jlK+Tl6F2lMCRkH8tAjqXVI+Es0mrdhUzk9gFamLs
cJASLUJWL1tKptHbaD2cKjkqEXj85HTLscDXLmUwpNbCVmdWBC8iiiqVsaqkbvcQ
46k7vqg5iAmP0DH6my6fy0KU0Ty5RT23XvQelywteO0rYvK6uzNRB+5E6rMd50yw
jx9VFpVYpoqtm+SIRakq6IoEBMhyNTC69DRWiJVymiSX/3EofCbeWNnrfbsIwA4m
VjdspGcTZ+q5Y3yzhCz5rztJhIy7rutpwXspRCyWTAgfuI1KovetGp84Pt36Q7oT
0LzAeOQrcCeuzMY/Htsw9+vgJGIGtOIKgmrBdtDp/qjHCsuCkNVP3rwJHrsqpuUo
bD12qZ+eO1P+tiZQkNQNr5s6hU56og3YJaE97Ywk/Wf0Kh8hSL4FtG+hNn5r2Ckv
R3H9HwilWfSRMtieQbsR
=C7oO
-----END PGP SIGNATURE-----

**Comment by rickysarraf** [comment]
To quote the same link:
I think the Chromium browser is doing what is documented in the developer docs. Because we have installed the extension before (and chromium may be storing that information somewhere). AFter this new policy file, we uninstall the extension. Thus not just it is removed, but also blacklisted.

> What if the user uninstalls the extension?
> 
> If the user uninstalls the extension through the UI, it will no longer be installed or updated on each startup. In other words, the external extension is blacklisted.
> 
> How do I get off the blacklist?
> 
> If the user uninstalls your extension, you should respect that decision. However, if you (the developer) accidentally uninstalled your extension through the UI, you can remove the blacklist tag by installing the extension normally through the UI, and then uninstalling it.
>

**Comment by nE0sIghT** [comment]
> I think the Chromium browser is doing what is documented in the developer docs

There is a collision because now we are using managed policies, not a "external installation".

From [docs](https://www.chromium.org/administrators/policy-list-3#ExtensionInstallForcelist):
>Specifies a list of apps and extensions that are installed silently, without user interaction, and which cannot be uninstalled by the user. All permissions requested by the apps/extensions are granted implicitly, without user interaction, including any additional permissions requested by future versions of the app/extension.
> ...
> This policy takes precedence over a potentially conflicting ExtensionsInstallBlacklist policy.

I think it may be a bug in Chromium

**Comment by rickysarraf** [comment]
![image](https://cloud.githubusercontent.com/assets/65480/18922917/1bac3cd4-85c8-11e6-808b-4c508bd22e4e.png)

Here's the screenshot I was able to capture.

**Comment by nE0sIghT** [comment]
Can you try modify policy file like this (add ExtensionInstallWhitelist)?

```
{
        "ExtensionInstallForcelist": [
                "gphhapmejobijbbhgpjhcjognlahblep;https://clients2.google.com/service/update2/crx"
        ],
        "ExtensionInstallWhitelist": [
                "gphhapmejobijbbhgpjhcjognlahblep"
        ]
}
```

**Comment by rickysarraf** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On Wed, 2016-09-28 at 09:43 -0700, Yuri Konotopov wrote:
> Can you try modify policy file like this (add ExtensionInstallWhitelist)?
> {
>         "ExtensionInstallForcelist": [
>                 "gphhapmejobijbbhgpjhcjognlahblep;https://clients2.google.com/
> service/update2/crx"
>         ],
>         "ExtensionInstallWhitelist": [
>                 "gphhapmejobijbbhgpjhcjognlahblep"
>         ]
> }

THanks Yuri. You nailed it. Adding a whitelist results in the extension showing
back again on the Chromium browser restart.


- -- 
Ritesh Raj Sarraf
RESEARCHUT - http://www.researchut.com
"Necessity is the mother of invention."
-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJX6/hhAAoJEKY6WKPy4XVp4K4P/37PdoOMW07XyxaM4+uvTTau
NMZMdRdhKU9/w+3QsarB+gOvSrlBDPNBOutYANnKHpSHpotNVPHFPdgOjm4huTyy
Muod32It7qnaAOcuVl5KdTymdgxBwW/TAUF7z08TDs6YcXj6y57sX7zgK1F8H6Gz
W78bADjuMsgF7HMQqFcv/GhT7B/gxRrFtlxepqT8X2E6vjeAOTBq76qH2D6X/kKf
0N3CvClK1Ylm/kNkVWdZTZgBxkc4kEi7MKJi/geLqvktYvwFTFxkrLfSjp7OOxzd
GDMxIjChvC+qRYm4MB0QTTOTYyyiEzn2TLY6zJ5wwyIqIGSya2B1Cu2E8XhKRbee
7FiTWUifhlEkl+jMc32AXGxT+q/Sd9Cow78sw+fX9xIXURVspuYtmKu4kc4cA2PP
GLrsOQqwViqf1LcMCnEHJ5DG1F2tVJyLKFcFj2JN7fOARc7sueX3hhKnkqWOKY91
DTQt6C3/e8g43rV325YhAz+1EeIZp6poE5mx3yfAUJKuwcn1oRBCcDaXr6KdQzvm
f4FT+t/y/9mfl85ZfgXEE+vVhB54KJoD8qc9Te7VKlA3y2tKcnpJwRiM7y0XlNzG
EdzirTwQrvb36iBSv7NzgioaexMkUgNXPjU2xoOYFG+VNDKcJ8TF/i7HCUN5yLDE
Y1uKM33yaeh+PH23k0Ka
=4cK/
-----END PGP SIGNATURE-----

**Comment by rickysarraf** [comment]
Okay! But the silent installation does not happen, as is documented in chrome docs. It seems to only be manually installable.

From docs:

> Specifies a list of apps and extensions that are installed silently, without user interaction, and which cannot be uninstalled by the user. All permissions requested by the apps/extensions are granted implicitly, without user interaction, including any additional permissions requested by future versions of the app/extension.
> ...
> This policy takes precedence over a potentially conflicting ExtensionsInstallBlacklist policy.

**Comment by nE0sIghT** [comment]
Ok, I think we found cause. I will try to reproduce it and will report this
to Chromium devs.

28 сент. 2016 г. 20:05 пользователь "Ritesh Raj Sarraf" <
notifications@github.com> написал:

**Comment by nE0sIghT** [comment]
Reported to Chromium devs: https://bugs.chromium.org/p/chromium/issues/detail?id=651197

**Comment by PeteLawler** [comment]
Hey guys, thanks for digging into this further. I'd not updated because I hadn't had a chance to test on a clean profile yet. Super appreciated that (a) you've done it for me and (b) it's confirmed I'm not losing my mind imagining this behaviour :)

**Comment by nE0sIghT** [comment]
@rickysarraf 

I just looked into [your debian package](http://ftp.debian.org/debian/pool/main/c/chrome-gnome-shell/) and found a typo in relax-chromium-policy.patch.
You missed comma before "ExtensionInstallWhitelist" so entire policy file became invalid json.

Despite the fact that this change "fixes" current issue :-) I think that ExtensionInstallWhitelist policy option do nothing for us.

**Comment by rickysarraf** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On Thu, 2016-09-29 at 22:56 -0700, Yuri Konotopov wrote:
> I just looked into your debian package and found a typo in relax-chromium-
> policy.patch.
> You missed comma before "ExtensionInstallWhitelist" so entire policy file
> became invalid json.

Oh! Thanks. I'll fix that.


> Despite the fact that this change "fixes" current issue :-) I think that
> ExtensionInstallWhitelist policy option do nothing for us.

Yes. I'm keeping it temporary, till the chromium bug can be triaged.

- -- 
Ritesh Raj Sarraf
RESEARCHUT - http://www.researchut.com
"Necessity is the mother of invention."
-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJX7iPWAAoJEKY6WKPy4XVpAzoQAJDfAl2eMG8gdbnk5Uxmar4D
8JRyPamvWeme7xngq0ctDqWwb45fvnYFfeUtwNjFTyMDvIaNqydx3HGfQTP/aMlv
lgQbUxCxWUjkMvIusPeqp+v8klN4zQVqaqq4rKhZWjUUsqF/L4wusMzShTVMMtzQ
gPxbMmziKVotzRe3CZ6Epm5xgza6LNarBNOo8Kk0vtlz800LOLpPvEV+IBhZIURk
4zJq80Ucn26ensmJ3wq+jDyNz0mNZGQ+OTGdOrH4eiANYGO6vhh9CYslC464XDiU
wKibtBI6ya94w1oia8RdTim1fsNV4cQU3scoh0VsEOFyJe34IC56CFkt+AZZX+eh
6DGC/AVY6i5ZmIC4yf4V9u3VmnEViEkFWo34B0/tEUzYlFsK/I1vkQQxBx5yh6kq
//gBHfvViBNodCnWfO9tGCaTCZt8q1MP0JGid3IyBYYrkEfTdb/5a3SbUsFjIRyi
0VZZfhbDlmSp40YyEF5pNhWKMQNPZGnqDQizSEBQOHo8OxMEdH8lOQRls3M5ozN1
5Q3JvF7Wd7TTX1HMLrlgbaXn7V//ZHm6vbl1rpUywO3m8B3xWjvcslR34ra1K6Am
kVF4oszwQjOFC4uxJOSaV3jlFMpEIcyhZ+fmO/tGm7s5JYJcJmza4P+98+g75b//
zbWfHfitmVoVkah5+2h2
=zGlH
-----END PGP SIGNATURE-----

**Comment by nE0sIghT** [comment]
This [issue is fixed](https://bugs.chromium.org/p/chromium/issues/detail?id=651197) is fixed in Chromium git master branch.

**Comment by rickysarraf** [comment]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

On Tue, 2016-10-11 at 09:01 -0700, Yuri Konotopov wrote:
> This issue is fixed is fixed in Chromium git master branch.

Yes. I saw that. Do you know which Chromium version that'd be part of ?
I'm going to hold off the policy file installation, until that version, for
Debian.


- -- 
Ritesh Raj Sarraf
RESEARCHUT - http://www.researchut.com
"Necessity is the mother of invention."
-----BEGIN PGP SIGNATURE-----

iQIcBAEBCgAGBQJX/RMZAAoJEKY6WKPy4XVpblgP/3gv58QUaPuVgrQdSxabDxPd
3QSyll4LkDBl4b1oLkfo9QIyigFh87rZlGnyHUQjC/SFDaBreU92mnneocxpdtEs
vsHHKKgaUstv9MfKnDHa4PX+J1EXqmMbvq8/c8XobmYlvicwYl2lOBZqRWbbepDI
fLXzgKqUzzSSIIGNNcEuWNfgtmY+sobbr7HmpbaJ6jVwa9aRZnwq9RC/aR8Sbi1B
JHk0Dk0apHB5Yl/2mOFdwJ9L8H+gLuqDzOCwH9UIQCbYVQiQIy+fs+pjBrFfdN7h
KLLYmqQOlCXVLnBzvYM0rYGqBThPbKU8KnnRG6S2JEqZGsbSAApFwGm6okXJbFVg
x0C7qhCNQDUQHbXO1p6V3MT1LFUDgovi3vB8rR3uBP/ZM7q2Wl4fP4LFZekIv+xE
sIqQsamcvLi1jEq/YB8oDVTaPR5Kbvjrwr/ArwwpRY8XonfUJ7lvE2sGw8nMLJ8q
dVyCCk7r+tdklj6q4nI72JtODdRkZcEeUZH0/Jet5j6BEuKtxFzR5dPe0WefCxi4
E+w6ZhqML3eF9F0oaianVC8Qv+9GzMAeMr4ZFVlvdhjsn+hc9LD8PzZSUWF2JY98
4NSsWP2l8r7adp0ag+ambVsigTkRAgODP/OQ0ofsLw5Wnt0X6KGFax+2tsOl1sP9
XYq7yuUX+XoOK7L+EBNv
=WPV6
-----END PGP SIGNATURE-----

**Comment by nE0sIghT** [comment]
> Do you know which Chromium version that'd be part of ?

It's unclear for now. I will monitor Chromium updates and will report when fix will land stable version.

**Comment by nE0sIghT** [comment]
Fixed in Chromium 56

**Comment by ierpe** [comment]
Hi,
I just updated a fresh jessie install to testing, and I have this problem...

Chromium Version : 57.0.2987.98 built on Debian 9.0, running on Debian 9.0 (64-bit)

I tried the previously mentioned solution but that didn't solve it for me :-(

**Comment by ierpe** [comment]
Ok found a solution, I had to add : 

CHROMIUM_FLAGS='--enable-remote-extensions'

to /etc/environment and logged out and logged in again, as found [here](https://productforums.google.com/forum/#!msg/chrome/8G6CZv7jVbw/HXKC8wpNBwAJ)

**Comment by jbicha** [comment]
@ierpe Debian's chromium package disables extensions by default. See Debian [#851927](https://bugs.debian.org/851927). This is controversial (there was a debian-devel discussion about it in late February).

**Comment by jbicha** [comment]
> I'm going to hold off the policy file installation, until that version, for
Debian.

@rickysarraf Since Chromium 57 is now in Debian testing, could you look into doing this now?

**Comment by rickysarraf** [comment]
@jbicha The policy file is already part of the package in Debian. And on my machine, Chromium is reflecting the correct status, as per policy.

![image](https://cloud.githubusercontent.com/assets/65480/24079578/16e30622-0cb1-11e7-9367-103ca700644b.png)

**Comment by jbicha** [comment]
@rickysarraf Great! Thanks for the screenshot. Yes, it's working here too.
